src/gentropy/datasource/finngen/finngen_finemapping.py

```diff
@@ -0,0 +1,286 @@
+# pylint: disable=unsubscriptable-object
+"""Datasource ingestion: FinnGen Finemapping results (SuSIE) to studyLocus object."""
```

Comment:
For consistency, I would rename the module to `/finngen/finemapping.py`. You are missing this part in the documentation `/docs/python_api/datasources/finemapping.md`